modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/GatlingCodegen.java

```diff
@@ -0,0 +1,359 @@
+                Set<Parameter> formParameters = new HashSet<>();
+                Set<Parameter> queryParameters = new HashSet<>();
+                Set<Parameter> pathParameters = new HashSet<>();
+
+                for (Parameter parameter : operation.getParameters()) {
+                    if (parameter.getIn().equalsIgnoreCase("header")) {
+                        headerParameters.add(parameter);
+                    }
+                    if (parameter.getIn().equalsIgnoreCase("formData")) {
+                        formParameters.add(parameter);
+                    }
+                    if (parameter.getIn().equalsIgnoreCase("query")) {
+                        queryParameters.add(parameter);
+                    }
+                    if (parameter.getIn().equalsIgnoreCase("path")) {
+                        pathParameters.add(parameter);
+                    }
+                    if (parameter.getIn().equalsIgnoreCase("body")) {
+                        BodyParameter bodyParameter = (BodyParameter) parameter;
+                        Model model = bodyParameter.getSchema();
+                        if (model instanceof RefModel) {
+                            String[] refArray = model.getReference().split("\\/");
+                            operation.setVendorExtension("x-gatling-body-object", refArray[refArray.length - 1] + ".toStringBody");
+                            Set<String> bodyFeederParams = new HashSet<>();
+                            Set<String> sessionBodyVars = new HashSet<>();
+                            for (Map.Entry<String, Model> modelEntry : swagger.getDefinitions().entrySet()) {
+                                if (refArray[refArray.length - 1].equalsIgnoreCase(modelEntry.getKey())) {
+                                    for (Map.Entry<String, Property> propertyEntry : modelEntry.getValue().getProperties().entrySet()) {
+                                        bodyFeederParams.add(propertyEntry.getKey());
+                                        sessionBodyVars.add("\"${" + propertyEntry.getKey() + "}\"");
+                                    }
+                                }
+                            }
+                            operation.setVendorExtension("x-gatling-body-feeder", operation.getOperationId() + "BodyFeeder");
+                            operation.setVendorExtension("x-gatling-body-feeder-params", StringUtils.join(sessionBodyVars, ","));
+                            try {
+                                FileUtils.writeStringToFile(new File(outputFolder + File.separator + dataFolder + File.separator + operation.getOperationId() + "-" + "BodyParams.csv"), StringUtils.join(bodyFeederParams, ","));
+                            } catch (IOException ioe) {
+                                LOGGER.error("Could not create feeder file for operationId" + operation.getOperationId(), ioe);
+                                System.exit(1);
```

Comment:
This should throw and let the CLI or wrapping application exit.

For example, if this is consumed outside of swagger-codegen-cli, you'll be terminating the application.

Reply:
Good point i will fix this later today